scripts/run-dask-process.sh

```diff
@@ -0,0 +1,245 @@
+   --ucx                   - initialize a ucx cluster with NVLink
+   --ucxib | --ucx-ib      - initialize a ucx cluster with IB+NVLink
+   -h | --help             - print this text
+
+ The cluster config order of precedence is any specification on the
+ command line (--tcp, --ucx, etc.) if provided, then the value of the
+ env var CLUSTER_CONFIG_TYPE if set, then the default value of tcp.
+
+"
+
+# CLUSTER_CONFIG_TYPE defaults to the env var value if set, else TCP
+CLUSTER_CONFIG_TYPE=${CLUSTER_CONFIG_TYPE:-TCP}
+START_SCHEDULER=0
+START_WORKERS=0
+
+if (( ${NUMARGS} == 0 )); then
+    echo "${HELP}"
+    exit 0
+else
+    if hasArg -h || hasArg --help; then
+        echo "${HELP}"
+        exit 0
+    fi
+    for a in ${ARGS}; do
+        if ! (echo " ${VALIDARGS} " | grep -q " ${a} "); then
+            echo "Invalid option: ${a}"
+            exit 1
+        fi
+    done
+fi
+
+if hasArg scheduler; then
+    START_SCHEDULER=1
+fi
+if hasArg workers; then
+    START_WORKERS=1
+fi
+# Allow the command line to take precedence
+if hasArg --tcp; then
+    CLUSTER_CONFIG_TYPE=TCP
+elif hasArg --ucx; then
+    CLUSTER_CONFIG_TYPE=UCX
+elif hasArg --ucxib || hasArg --ucx-ib; then
+    CLUSTER_CONFIG_TYPE=UCXIB
+fi
+
+########################################
+
+#export DASK_LOGGING__DISTRIBUTED="DEBUG"
+
+#ulimit -n 100000
+
+SCHEDULER_LOG=${LOGS_DIR}/scheduler_log.txt
+WORKERS_LOG=${LOGS_DIR}/worker-${HOSTNAME}_log.txt
+
+
+function buildTcpArgs {
+    export DASK_DISTRIBUTED__COMM__TIMEOUTS__CONNECT="100s"
+    export DASK_DISTRIBUTED__COMM__TIMEOUTS__TCP="600s"
+    export DASK_DISTRIBUTED__COMM__RETRY__DELAY__MIN="1s"
+    export DASK_DISTRIBUTED__COMM__RETRY__DELAY__MAX="60s"
+    export DASK_DISTRIBUTED__WORKER__MEMORY__Terminate="False"
+
+    SCHEDULER_ARGS="--protocol=tcp
+                    --port=$DASK_SCHEDULER_PORT
+                    --scheduler-file $SCHEDULER_FILE
+                "
+
+    WORKER_ARGS="--rmm-pool-size=$WORKER_RMM_POOL_SIZE
+             --local-directory=/tmp/$LOGNAME
+             --scheduler-file=$SCHEDULER_FILE
+             --memory-limit=$DASK_HOST_MEMORY_LIMIT
+             --device-memory-limit=$DASK_DEVICE_MEMORY_LIMIT
+            "
+
+}
+
+function buildUCXWithInfinibandArgs {
+
+    export UCX_MAX_RNDV_RAILS=1
+    export UCX_MEMTYPE_REG_WHOLE_ALLOC_TYPES=cuda
+    export DASK_RMM__POOL_SIZE=0.5GB
+    export DASK_DISTRIBUTED__COMM__UCX__CREATE_CUDA_CONTEXT=True
+
+    SCHEDULER_ARGS="--protocol=ucx
+                --port=$DASK_SCHEDULER_PORT
+                --interface=$DASK_CUDA_INTERFACE
+                --scheduler-file $SCHEDULER_FILE
+               "
+
+    WORKER_ARGS="--interface=$DASK_CUDA_INTERFACE
+                --rmm-pool-size=$WORKER_RMM_POOL_SIZE
+                --local-directory=/tmp/$LOGNAME
+                --scheduler-file=$SCHEDULER_FILE
+                --memory-limit=$DASK_HOST_MEMORY_LIMIT
+                --device-memory-limit=$DASK_DEVICE_MEMORY_LIMIT
+                "
```

Comment:
We should also add `jit-unspill`
```suggestion
                --jit-unspill
                "
```